lib/gatling/init_script_template.sh.eex

```diff
@@ -13,6 +13,6 @@ export MIX_ENV=prod
 export PORT=<%= @port %>
 export HOME=<%= System.user_home %>/deployments/<%= @project_name %>
 
-export $(cat <%= System.user_home %>/deployments/<%= @project_name %>/.env)
+[ -f $HOME/.env ] && export $(cat $HOME/.env)
```

Comment:
Ok, I see – but I think we should not overwrite the `$HOME` variable provided by the system. How about `$PROJECT_DEPLOY_DIR` or something like that?

Maybe we should align this with the paths and dirs set in Utilities.